### PR TITLE
Extract pension pot and options overview sections

### PR DESCRIPTION
**Pull request by gjvis**

### CODE REVIEW

**Comment by stevenwilkin** [comment]
:+1:

File: app/templates/output_document_options_overview.html.erb [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        the options we talked about in the appointment.</p>
+
+      <p class="emphasize l-emphasize-gap">This is just a brief <br />
+        reminder of the options we discussed. You can find detailed information at
+        <a href="https://www.pensionwise.gov.uk/pension-pot-options">
+          pensionwise.gov.uk/pension-pot-options</a></p>
+
+      <p class="emphasize">
+        Whichever option you decide on ask your provider about fees and charges.
+        Also make sure you know how much<br /> tax you pay  - go to
```
Comment (andrewgarner) [inline app/templates/output_document_options_overview.html.erb]: Why space after the line break?

Double space before the dash?

File: app/templates/output_document_options_overview.html.erb [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        (take flexible lump sums or get a flexible income)</span></h3>
+
+      <p>You can take your money flexibly in 2 ways. It depends what kind of
+        income you want to get from your pot. You can always take 25% tax
+        free.</p>
+
+      <p><b>Take flexible lump sums:</b><br />
+        Your pot stays where it is and you can
+        take money out when you want to. 25% of each cash amount you take out is
+        tax free, the rest is taxable. This is also known as 'uncrystallised
```
Comment (andrewgarner) [inline app/templates/output_document_options_overview.html.erb]: We're using `'` and `‘` `’` interchangeably.

File: app/templates/output_document_pension_pot.html.erb [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+          retirement.
+        </p>
+      <% elsif output_document.income_in_retirement == 'other' %>
+        <p>
+          You told us that you have other income in retirement, eg from a
+          defined benefit (final salary) pension, savings, rental income etc. 
+        </p>
+      <% end %>
+
+      <p>Most people are entitled to the State Pension.  You can apply for a State
```
Comment (andrewgarner) [inline app/templates/output_document_pension_pot.html.erb]: Double space `Pension.  You`

**Comment by andrewgarner** [comment]
:-1:

**Comment by gjvis** [comment]
This is pulled in from the other repo. If there are issues then they also exist in the PDFs that we are currently sending out to people.

**Comment by gjvis** [comment]
Changes need to happen there, and then get pulled across to this repo, or we'll break the ability to cherry-pick. (PDF_Output "owns" these files)

**Comment by andrewgarner** [comment]
Fair enough. Do you want me to raise them as issues?

**Comment by gjvis** [comment]
Nah, I'm reviewing them with design and content now to make sure they aren't intentional. I'll fix in the other repo and pull them across.

**Pull request merged**